xalia/Win32/NonclientScrollProvider.cs

```diff
@@ -25,19 +25,56 @@ public NonclientScrollProvider(HwndProvider hwndProvider, UiDomElement element,
             { "y", "win32_y" },
             { "width", "win32_width" },
             { "height", "win32_height" },
+            { "state", "win32_state" },
         };
 
         public bool Vertical { get; }
 
-        private bool _watchingLocation;
-        private bool _locationKnown;
-        private int _sbiChangeCount;
+        const int SBI_LOCATION = 0;
+        const int SBI_STATE = 1;
+        const int SBI_NUM_CONSTANTS = 2;
+

```

Comment:
Would these be better as an `enum`? Functionally it makes no difference, but it'd avoid having to update `SBI_NUM_CONSTANTS` if you ever add another one.

Reply:
Maybe. The disadvantage is that enums need an explicit cast to int.

Reply:
Fair enough. :)